bundles/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/type/HomematicTypeGeneratorImpl.java

```diff
@@ -237,7 +236,8 @@ private ThingType createThingType(HmDevice device, List<ChannelGroupType> groupT
 
         List<ChannelGroupDefinition> groupDefinitions = new ArrayList<>();
         for (ChannelGroupType groupType : groupTypes) {
-            String id = StringUtils.substringAfterLast(groupType.getUID().getId(), "_");
+            int usPos = groupType.getUID().getId().indexOf("_");
```

Comment:
I don't know if it matters in the case. But shouldn't this be:
```suggestion
            int usPos = groupType.getUID().getId().lastIndexOf("_");
```

Reply:
I am not absolutely sure whether the ID can contain more than one "_". Therefore, you are right and lastIndexOf is definitely better